addons/godot-next/inspector_plugins/behavior_collection_inspector_plugin.gd

```diff
@@ -0,0 +1,91 @@
+tool
+extends EditorInspectorPlugin
+
+##### PROPERTIES #####
+
+var class_type: ClassType
+
+##### NOTIFICATIONS #####
+
+func _init():
+	class_type = ClassType.new()
+
+##### VIRTUALS #####
+
+func can_handle(object):
+	return object is BehaviorCollection
+
+func parse_property(object, type, path, hint, hint_text, usage):
+	match type:
```

Comment:
These two match segments execute the same behavior. You can catch both cases by simply using a comma between the two case values.

    TYPE_ARRAY, TYPE_DICTIONARY:
        if hint == ......